library/globals.inc.php

```diff
@@ -2862,7 +2867,12 @@ function gblTimeZones()
             '0',                              // default
             xl('Enable logging of CDR Engine Queries.') . ' (' . xl('Note that Audit Logging needs to be enabled above') . ')'
         ),
-
+        'audit_events_http-request' => array(
+            xl('Audit Logging user page history.'),
+            'bool',                           // data type
+            '0',                              // default = false for backwards compatibility
```

Comment:
Any reason not to make this on by default. Otherwise it will never be used :)

Reply:
Wasn't sure if we wanted to add a ton of logs for older installations.  But I can make this default.  Because of the way the background services works it will add a lot of logs for those requests but I think that's preferable to not logging at all.  I actually had the thought to add in a unique request id also that would allow grouping entire page request histories into a single log trail.

Reply:
That would be neat to group them by session. Check out SessionTracker where a session marker is stored in database and as a session variable (added the mechanism for a more robust way to track session expirations).

Reply:
Realized that mine above would be a a login session and looks like yours would be by request grouping which would also be neat. They both could be helpful when auditing.

Reply:
Made default, will push request id / session id logging to a future update.